dubbo-registry/dubbo-registry-multiple/src/test/java/org/apache/dubbo/registry/multiple/MultipleRegistry2S2RTest.java

```diff
@@ -144,8 +145,7 @@ public void testSubscription() throws InterruptedException {
 
         String path = "/dubbo/" + SERVICE2_NAME + "/providers";
         List<String> providerList = zookeeperClient.getChildren(path);
-        Assertions.assertTrue(!providerList.isEmpty());
-        System.out.println(providerList.get(0));
```

Comment:
System.out is useless in unit test. So remove this.

Reply:
ok